### PR TITLE
Migrate Configs to pyproject.toml (PEP518)

### PR DESCRIPTION
**Pull request by nlsdfnbch**
I got rid of the setup.cfg and setup.py files, switching from `setuptools` to `flint`.

Added a mypy.ini (no thanks to Guido for not merging
a PR implementing support for PEP518).

Also added some hypothetical environments in tox.ini (docs, tests).

Lint env already runs (yay) with errors (oh).

----------------------------------------

Disclaimer: I have no clue on how mypy's innards work or how it's configured (I'm a pyre guy) - if my config's shit, let me know!

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[tox]
+envlist = py37
+isolated_build = True
+
+[tox:.package]
+basepython = 3
+
+[testenv:unittests]
```
Comment (ulope) [inline tox.ini]: I'd change this to just `[testenv]` that way it's the default when simply running `tox` without any arguments.

**Comment by nlsdfnbch** [comment]
> lgtm.
> 
> One suggestion below.

Did as suggested, updated tox.ini and config accordingly!

File: pyproject.toml [hunk 2]
```diff
@@ -1,3 +1,95 @@
...
+
+######################
+# FLIT CONFIGURATION #
+######################
+
+[tool.flit.metadata]
+# General Meta data
+dist-name = "raiden-scenario-player"
+module = "scenario_player"
+author = "Ulrich Petri"
```
Comment (ulope) [inline pyproject.toml]: Thanks, but this should probably be `Brainbot Labs Est.` ;)

File: pyproject.toml [hunk 3]
```diff
@@ -1,3 +1,95 @@
...
+author-email = "noreply@brainbot.com"
```
Comment (ulope) [inline pyproject.toml]: In Raiden we use `contact@brainbot.li`

File: pyproject.toml [hunk 4]
```diff
@@ -1,3 +1,95 @@
...
+maintainer = "Nils Diefenbach"
```
Comment (ulope) [inline pyproject.toml]: Same as with author.

**Comment by nlsdfnbch** [comment]
Force merging this, as I mereley updated meta data as requested.

**Pull request merged**